### PR TITLE
remove unused scripts, references to docker registry, and update readme

### PR DESCRIPTION
**Pull request by hillalex**
* Removes unused scripts and references to the private docker registry
* Updates README
* Adds release script tests to the BuildKite pipeline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,21 +20,11 @@ v1.23.0 which appends random strings to container names each time they run.
...
    (e.g. https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-16-04.) 
    You may need to restart your machine for group changes to take effect.
-1. Configure your Docker client to use our registry by following instructions 
-   here: https://github.com/vimc/montagu-registry/tree/master#login 
-   
-   This is where you will need to have Vault 
-   installed, and the VAULT_ADDR variable specified, e.g. by adding 
-   `export VAULT_ADDR='https://support.montagu.dide.ic.ac.uk:8200'` to your profile using ` sudo nano ~/.profile` 
 1. Install node dependencies by running `npm install --prefix src/app/static`. Javascript and CSS will be compiled automatically as part of the gradle build.
+1. For local development and testing run all dependencies (Orderly Server etc.) with `./dev/run-dependencies.sh`.
```
Comment (EmmaLRussell) [inline README.md:24]: While we're housekeeping, it's probably worth adding here that you'll also want to run the add-test-users script in order to have access to the demo reports (or maybe just roll running that script into dev/run-dependencies?) and mention the login details.

**Review by EmmaLRussell — COMMENTED** [review]
Good stuff. Looks like there's still a reference to the private registry in `scripts/test-cli.sh`? Do we still need that?

File: README.md (hunk 1 above)
Comment (hillalex) [reply README.md:24]: Well spotted, thanks

**Comment by hillalex** [comment]
> Good stuff. Looks like there's still a reference to the private registry in `scripts/test-cli.sh`? Do we still need that?

That script is referenced in the userCLI readme as being a way to test it manually, so I guess no harm in keeping it. I've updated the private registry reference.

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**